mathics/builtin/inout.py

```diff
@@ -1820,6 +1820,7 @@ def apply_tex(self, expr, evaluation):
             # Replace multiple newlines by a single one e.g. between asy-blocks
             tex = MULTI_NEWLINE_RE.sub('\n', tex)
 
+
```

Comment:
there are a few whitespace changes that don't need to be included.

Reply:
Done